standards/code-style.adoc

```diff
@@ -648,26 +650,28 @@ or `Fragment`.
 
 In the case of Activities the method is usually called `newStartIntent()`:
 
-[source, java]
+[source, kotlin]
 ----
-public static Intent newStartIntent(Context context, User user) {
-	Intent intent = new Intent(context, ThisActivity.class);
-	intent.putParcelableExtra(EXTRA_USER, user);
-	return intent;
+fun newStartIntent(context: Context, user: User): Intent {
```

Comment:
Technically this should be in a companion object to mirror the static functionality of the Java example (like you did below).